.github/workflows/ci.yml

```diff
@@ -13,12 +13,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [18.x, 20.x]
```

Comment:
We can probably ditch `18.x` here and test against 20/22 instead as we don't need to support 18.